dask/utils.py

```diff
@@ -406,12 +406,20 @@ def dispatch(self, cls):
                 return lk[cls2]
         raise TypeError("No dispatch for {0}".format(cls))
 
-    def __call__(self, arg):
+    def __call__(self, arg, *args, **kwargs):
         """
         Call the corresponding method based on type of argument.
         """
         meth = self.dispatch(type(arg))
-        return meth(arg)
+        return meth(arg, *args, **kwargs)
+
+    @property
+    def __doc__(self):
+        try:
+            func = self.dispatch(object)
+            return func.__doc__
+        except TypeError:
+            return "Single Dispatch for %s" % self.__name__
```

Comment:
I'd just return `""`, as this is the default docstring for all classes/functions in python.

Reply:
Oop, nevermind, I can't read. Thought you were raising an error here, this is fine as is.